### DISCUSSION TITLE
Detail Document how to use

### DISCUSSION THREAD

**Discussion by topninja** [Atlantis-Lab/tinkoff-api#54] [General]
looks like good package that updates recently
but I am not sure how to use this in detail.
const response = await tinkoff.api.autopayments.init({ ... })
should I check code for use this?

**Comment by TorinAsakura** [comment]
@topninja hello Lyu.
I highly recommend you check code anyway cause it's usefull

**Comment by TorinAsakura** [comment]
@topninja we'll update docs today

**Comment by TorinAsakura** [comment]
@topninja https://github.com/Atlantis-Lab/tinkoff-api/pull/55

**Comment by TorinAsakura** [comment]
> looks like good package that updates recently
> but I am not sure how to use this in detail.
> const response = await tinkoff.api.autopayments.init({ ... })
> should I check code for use this?

as a thank you can simply give a stargazer on repo)
